packages/tempus/src/Tempus_StepperFactory_impl.hpp

```diff
@@ -1616,7 +1616,6 @@ setStepperSolverValues(
   auto subPL = sublist(defaultSolverPL, "NOX");
   solver->setParameterList(subPL);
   if (stepperPL != Teuchos::null) {
-    std::cout << "stepperPL = \n" << *stepperPL << "\n" << std::endl;
```

Comment:
I think we write out the solver ParameterList for DIR/IMEX. Can we just remove that too?

Reply:
This was a stray std::cout that should not there.  I think it was leftover from some debugging.  I could not find any std::cout from DIRK/IMEX.  Can you point me to where you think the write is?